pkg/apiserver/openapi/openapi.go

```diff
@@ -29,7 +29,21 @@ import (
 
 var verbs = util.CreateTrie([]string{"get", "log", "read", "replace", "patch", "delete", "deletecollection", "watch", "connect", "proxy", "list", "create", "patch"})
 
-// ToValidOperationID makes an string a valid op ID (e.g. removing punctuations and whitespaces and make it camel case)
+// forcePrefixForOpIDs contains list of operation IDs that will get a prefix (usually GroupVersion) to prevent duplicates.
+var forcePrefixForOpIDs = map[string]bool{
+	"getAPIResources": true,
+	"getAPIGroup":     true,
+	"getAPIVersions":  true,
+}
+
+// forcePrefixForTags contains list of tags which their Operation IDs will get a prefix (usually GroupVersion) to prevent duplicates.
+// It is preferred to put alpha/beta (e.g., *_v1beta1) tags in this list rather than released tags (e.g. CoreV1).
+// Tags are usually lowercase group or group_version
+var forcePrefixForTags = map[string]bool{
```

Comment:
This doesn't belong in this package - this package should not know anything about specific versions.  You must pass this in.